scripts/lint_jsonnet.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# NOTE: This script expects to be run from the project root with
+# ./scripts/lint_jsonnet.sh
+
+failures=0
+while read file; do 
+  jsonnet fmt --test -i $file
+  if [ $? -ne 0 ]; then
+    echo "'$file' failed Jsonnet format check. Run 'jsonnet fmt' to fix."
```

Comment:
```suggestion
    echo "'$file' failed Jsonnet format check. Run 'jsonnet fmt <filename> -o <filename>' to fix."
```
Since it outputs to stdout by default?
Or I guess the lint script to just fmt anyway?

Reply:
jsonnet fmt has an `-i` flag to do the formatting in place. I've updated the script so the reformat command is copy-pastable.